### ISSUE TITLE
tflite 模型转换 正确性校验出错 TESTERROR 

### ISSUE THREAD

**Issue by Zepan** [alibaba/MNN#1710]
你好，测试一个非常简单的纯卷积堆叠的tflite转换，发现部署结果不正确，使用 fastTestTflite.py 自测发现误差非常大，
本来还怀疑是不是tf1.15版本太老不兼容，但是用tf1.15转mobilenet是自测通过的。
请帮忙排查下，照理来说这种简单结构不应该出问题才对。。

模型下载地址：
链接：https://pan.baidu.com/s/1BUF6izmuB8R_SX5yhHLVPg 
提取码：tpm4 

自测日志：
~~~
INFO: Initialized TensorFlow Lite runtime.
tflite/test.tflite
['SoftmaxLayer/softmax/Softmax']
inputs:
the_inputs
outputs:
tflite/SoftmaxLayer/softmax/Softmax.txt
Test tflite
Start to Convert Other Model Format To MNN Model...
Start to Optimize the MNN Net...
inputTensors : [ the_inputs, ]
outputTensors: [ SoftmaxLayer/softmax/Softmax, ]
Converted Success!
the_inputs
output: SoftmaxLayer/softmax/Softmax
SoftmaxLayer/softmax/Softmax: (1, 16, 1250, )
TESTERROR SoftmaxLayer/softmax/Softmax value error : absMaxV:0.999892 - DiffMax 0.747764
Save mnn result to  .error director
~~~

**Comment by JiaoPaner** [comment]
请问解决了吗 我也遇到这个问题

**Comment by jxt1234** [comment]
该模型中有 depth_multier ，目前 MNN 不支持 tflite 的 depth_multier （pb 的支持）。
对于此模型用特例方式解决了，后续再考虑支持 tflite 的 depth_multier
